actioncable/app/assets/javascripts/action_cable/connection.coffee

```diff
@@ -46,12 +51,21 @@ class ActionCable.Connection
     else
       @open()
 
+  getProtocol: ->
+    @webSocket?.protocol
+
+  isProtocolSupported: ->
+    @getProtocol() in protocols and @getProtocol() isnt unsupported_protocol
```

Comment:
We don't. I just thought it would be clearer to any future maintainers that this was a special case, but at the same time not get accidentally removed.